src/apps/slackapp/slackapp/bolt_app.py

```diff
@@ -107,7 +106,6 @@ def get_response(
         verbose_logger (BaseVerboseLogger): verbose logger to be used
         bot_info (Dict[str, str]): information of the Slack bot
         llm (SherpaChatOpenAI, optional): LLM to be used. Defaults to None.
-        team_id (str, optional): team id of the Slack workspace. Defaults to "".
         user_id (str, optional): user id of the Slack user. Defaults to "".
```

Comment:
Update the `user_id` comment to explain this is actually team ID combined with user ID, so that it can be a globally unique value for the usage tracker.